downstream/modules/platform/con-edge-manager-drop-dir.adoc

```diff
@@ -0,0 +1,12 @@
+[id="edge-manager-drop-dir"]
+
+= Drop-in directories
+
+Use drop-in directories to add, replace, or remove configuration files that the service aggregates.
+Do not directly edit your configuration files that might cause deviation from the target configuration.
```

Comment:
Suggest changing to : Do not directly edit your configuration files because it can cause deviations from the target configuration.